userpanel/style/bclean/body.html

```diff
@@ -1,7 +1,7 @@
 <div class="row justify-content-end no-gutters">
 	<div>
 		<small>
-			{trans("Session expires after:")}
+			{trans("Logged as:")} {$userinfo.customername|truncate:60:"... "}, {trans("Session expires after:")}
```

Comment:
a tu nie powinno być jeśli już " ..." zamiast "... ". Czemu służy spacja na końcu "... "?

Reply:
Patrzyłeś jak to wizualnie wygląda? Jeśli jest bez końcowej spacji to zlewa się z przecinkiem.

Reply:
A nie lepiej informację o zalogowanym dać pod czasem do końca sesji?

Reply:
Puść commita, żebym to mógł oczyma.

Reply:
@chilek pokaż co masz na myśli.

Reply:
jeden wiersz - to co jest - czas do końca sesji
drugi wiersz - informacja o zalogowanym.

Reply:
Odwrotnie lepiej wygląda bo na górze jest dłuższy wiersz. 
Dodałem poprawkę.